mz_bokeh_package/components/plot_settings.py

```diff
@@ -180,6 +180,11 @@ def __init__(
         self._included_settings = included_settings or BASE_SETTINGS
         self._plot_tool_description = "Plot Settings"
 
+        # List of renderers to exclude (currently, only trend_line):
+        self._excluded_renderers = ["trend_line",
+                                    "upper_prediction_limit_intercept",
+                                    "lower_prediction_limit_intercept"]
```

Comment:
As mentioned in the other PR, this feels like a bad idea. The purpose of this package is to provide a general structure that is accessible to all apps. Hard-coding a very specific application into it, has many risks, just to name two: Either you will keep adding stuff to this list and bloat it, or you'll use these names in other apps to get the desired functionality, but with a different purpose. Then, communicating between package by these secretive codes has all risks of typos, and results in a lot of problems in identifying errors.